packages/table/test/cellTestUtils.ts

```diff
@@ -21,17 +21,17 @@ export const CellType = {
 export function expectCellLoading(cell: Element, cellType: CellType, loading = true) {
     if (loading) {
         expect(cell.classList.contains(Classes.LOADING)).to.be.true;
-        expect(cell.querySelector(`.${Classes.SKELETON}`)).to.not.be.null;
         if (cellType !== CellType.BODY_CELL) {
             const headerNameText = cellType === CellType.COLUMN_HEADER
                 ? cell.querySelector(".bp-table-column-name-text")
                 : cell.querySelector(".bp-table-row-name");
             expect(headerNameText).to.not.be.null;
             expect(headerNameText.textContent).to.equal("");
             expect(headerNameText.children.length).to.equal(1);
+            expect(headerNameText.children[0].classList.contains(Classes.SKELETON)).to.be.true;
         } else {
             expect(cell.children.length).to.equal(1);
-            expect(cell.querySelector(`.${Classes.SKELETON}`)).to.not.be.null;
+            expect(cell.children[0].classList.contains(Classes.SKELETON)).to.not.be.null;
```

Comment:
wait why back to this format?
did my comments lead us in a circle? 😶 

you're testing too much here and it's not helping. 
it should be sufficient to assert that a selector exists, for example:
`.bp-table-row-name .pt-skeleton`